update_member_clt.py

```diff
@@ -0,0 +1,20 @@
+from config import settings
+from models import Member
+
+
+def main():
+    print("Type the acronyms of the members who are CLT now.")
+    print("Example: 'JNR LAB FSN PYC'")
+    acronyms = input("").upper().split()
```

Comment:
Old_timesheet_migration is based in lower case pattern. Isn't better make this script in lower case pattern too?

Reply:
Ohh you are right, will change this right now! I still need to change this on the production database.